src/ctype/CTypeUtils.ts

```diff
@@ -42,3 +42,13 @@ export function verifyClaimStructure(claim: any, schema: any): boolean {
 export function getHashForSchema(schema: ICType['schema']): string {
   return Crypto.hashObjectAsStr(schema)
 }
+
+export function validateSchema(
+  schema: object,
+  validate: object
+): boolean | PromiseLike<any> {
+  const ajv = new Ajv()
+  ajv.addMetaSchema(CTypeModel)
+  const result = ajv.validate(schema, validate)
+  return result
+}
```

Comment:
I have added this function because of the flow of validating. I see that this function will be specifically for the quote and checking the validity of specific quotes against the metaschema quote. 

Additionally, I believe this needs cleaning validating from Ajv schema needs cleaning up and each should be a helper function for Claim, CType, Offer, etc. As each has specific uses and some may need to be compiled. But this is out of the scope of the ticket.